modin/pandas/dataframe.py

```diff
@@ -3169,3 +3169,6 @@ def __reduce__(self):
         return self._inflate_light, (self._query_compiler, pid)
 
     # Persistance support methods - END
+
+    # Namespace for experimental functions
+    modin = CachedAccessor("modin", ExperimentalFunctions)
```

Comment:
```suggestion
    _exp = CachedAccessor("_exp", ExperimentalFunctions)
```
I thought we agreed on _exp. I would prefer to use it instead of `modin` attribute, which is a bit vague to me.

Reply:
It seemed to me that this was an open question, judging by what I wrote here https://github.com/modin-project/modin/pull/6764#issuecomment-1832028750.

Using private attributes can eliminate the desire to try, which is why I thought about naming the attribute with a public name, and to make it clear that the attribute is not implemented in pandas.

Reply:
Makes sense, let's leave as is.